### ISSUE TITLE
Add dotnet-monitor to appservice images

### ISSUE THREAD

**Issue by balag0** [Azure/azure-functions-docker#358]
To keep things in sync with other app service images. 

Will wait for the changes to be implemented in appservice images before starting work on this.

**Comment by vivekjilla** [comment]
This internal task was gated on other images and is no longer applicable, with no activity since November 30, 2020 (about 6 years). We're closing it. Please open a new issue if dotnet-monitor parity is still wanted on a current image.
